docs/meshcloud.policies.md

```diff
@@ -41,15 +82,15 @@ You may encounter policies in the meshPanel when doing any of the following acti
 
 It might happen, consciously or unconsciously, that you violate one or more policies. At every place in the meshPanel that is mentioned above this section, we prevent you from finalizing the violation and you will be prompted with an error message explaining which meshPolicy or policies you violated, and why.
 
-Take this policy violation as an example (see the picture below), where we have a meshPolicy defined on 'meshCustomer' and 'meshProject', both on the `environment` tag.
+Take this meshPolicy violation as an example (see the picture below), where we have a meshPolicy defined on 'meshCustomer' and 'meshProject' with the evaluation strategy 'Subset', both on the `environment` tag.
 
 ![Example Policy Error Message](assets/mesh_policies/policies_example_error_message.png)
 
 The project that is being created called 'my-example-project-prod' wants environment `prod` and the meshCustomer 'managed-customer' the project is being created in, has environments `dev`, `test` and `qa` defined. This means there is a mismatch as `<prod>` is not inside `<dev, test, qa>`. In order to solve this problem, we have to pick an environment that is defined on the meshCustomer, e.g. `dev`. After picking a valid environment value, we can save the project again, and (if we don't violate any further policies) the project is successfully created, including the right compliance for your organization! ✅
 
 ### Are there any other places where policies are enforced?
 
-Besides end-users being impacted at the places above, there are also other places where policy violations could be caused. The diagram below describes all possible relationships between meshObjects and the behavior that is expected, depending on who is doing the change.
+Besides end-users being impacted at the places above, there are also other places where meshPolicy violations could be caused. The diagram below describes all possible relationships between meshSubjects and the behavior that is expected, depending on who is doing the change.
```

Comment:
Added mermaid graph.

Reply:
```
flowchart TB
    subgraph Legend
    start1["blocking violationon new assignments"]:::legendClass --> stop1[ ]:::legendClass
    start2["logging violation but not blocking"]:::legendClass -.-> stop2[ ]:::legendClass
    start3["blocks on creation and editing"]:::legendClass --> stop3[ ]:::legendClass
    end
    subgraph In case of policy violation
      meshCustomer:::authorativeClass --> meshProject
		  meshProject:::authorativeClass -.-> meshCustomer
		  meshProject --> meshLandingZone
		  meshLandingZone -.-> meshProject
		  meshUser/Group --> meshCustomer
		  meshCustomer -.-> meshUser/Group
		  meshUser/Group -.-> meshProject
		  meshProject --> meshUser/Group
    end

classDef authorativeClass fill:#dae8fc;
classDef legendClass height:0px;
linkStyle 2 stroke: blue;
linkStyle 3 stroke: blue;```

![image](https://user-images.githubusercontent.com/64793818/160790445-95c48160-e345-4249-8438-651da2bf0202.png)

Reply:
Added.